pages/pricing/_subscription-tabs.html

```diff
@@ -0,0 +1,105 @@
+      <div class="tab-pane active" id="tab-gruntwork-subscription" role="tabpanel">
+        <div class="row">
+          <div class="col-xs-12 col-md-6">
+            <div class="panel panel-default panel-dark">
+              <div class="panel-body text-center">
+                <h3 style="margin-bottom: 10px;">{{ site.data.pricing.tier1.title }}</h3>
+                <p class="big-price"><span>$</span>{{ site.data.pricing.tier1.price[0] }}</p>
+                <h6 class="help-block" style="margin-bottom: 0;">{{ site.data.pricing.tier1.details }}</h6>
+                <p class="text-muted small">{{ site.data.pricing.tier1.description }}</p>
+                <p style="margin-bottom: 10px;"><a href="/contact" class="btn btn-info" role="button">Get Started</a></p>
+                {% comment %}<p class="text-muted small">Annual contract, billed monthly.</p>{% endcomment %}
+              </div>
+            </div>
+          </div>
+          <div class="col-xs-12 col-md-6">
+            <div class="panel panel-default panel-dark">
+              <div class="panel-body text-center">
+                <h3 style="margin-bottom: 10px;">{{ site.data.pricing.tier2.title }}</h3>
+                <p class="big-price"><span>+ $</span>{{ site.data.pricing.tier2.price[0] }}</p>
+                <h6 class="help-block" style="margin-bottom: 0;">{{ site.data.pricing.tier2.details }}</h6>
+                <p class="text-muted small">{{ site.data.pricing.tier2.description }}</p>
+                <p style="margin-bottom: 10px;"><a href="/contact" class="btn btn-info" role="button">Get Started</a></p>
+                {% comment %}<p class="text-muted small">Annual contract, billed monthly.</p>{% endcomment %}
+              </div>
+            </div>
+          </div>
+          <div class="col-xs-12">
+            <div class="panel panel-default panel-dark">
+              <div class="panel-body text-center">
+                <h3 style="margin-bottom: 10px;">{{ site.data.pricing.tier3.title }}</h3>
+                <p class="big-price"><span>+ $</span>{{ site.data.pricing.tier3.price }}</p>
+                <h6 class="help-block" style="margin-bottom: 0;">{{ site.data.pricing.tier3.details }}</h6>
+                <p class="text-muted small">{{ site.data.pricing.tier3.description }}</p>
+                <p style="margin-bottom: 0;"><a href="/contact" class="btn btn-info" role="button">Get Started</a></p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <p class="text-muted text-center small" style="margin-bottom: 0; text-decoration: underline">A user is any human or machine user that makes use of Gruntwork's services</p>
+        <p class="text-muted text-center small" style="margin-bottom: 0;">*Gruntwork Subscription requires annual contract, Dedicated Support is month-to-month.</p>
+        <img class="grunty-confident" src="/assets/img/grunty-pricing.png" width="238" height="288" alt="Grunty confident">
```

Comment:
We tried this, but it added too much extra space between the page title and pricing tables, and it looked somewhat cheesy. Since our pricing panels lost some height, I think Grunty shows high enough to be visible in a screen-fold with the rest of the pricing. Let us know if you still want us to do this, we felt it was better otherwise. The attached screenshot is for 1440x900 viewport sizes. 
<img width="1360" alt="screenshot 2018-05-01 19 26 12" src="https://user-images.githubusercontent.com/17956309/39502536-a4713300-4d75-11e8-8da1-be901efe81bc.png">